indexer-app/src/cmr/indexer/data/index_set.clj

```diff
@@ -333,6 +333,7 @@
           :has-formats (m/stored m/bool-field-mapping)
           :has-transforms (m/stored m/bool-field-mapping)
           :has-spatial-subsetting (m/stored m/bool-field-mapping)
+          :has-opendap-url m/bool-field-mapping
```

Comment:
I'd just go ahead and store it now rather than adding another field for the inevitable ticket that someone wants this information returned in the JSON response. A boolean for 50K collections takes almost no space when compared to any granule field.